db/migrate/20240318080818_drop_associations_table.rb

```diff
@@ -0,0 +1,10 @@
+class DropAssociationsTable < ActiveRecord::Migration[7.1]
+  def change
+    drop_table :tag_associations do |t|
```

Comment:
Might be useful to drop index separately in case one wants to reverse this migration.

Reply:
Yes, you can reverse it, but the result won't be the same. You can compare the schema before and after reversing.

Reply:
I've confirmed that is not the case, so it needs to be added.